minishift/README.md

```diff
@@ -93,4 +100,4 @@ eval $(minishift docker-env)
 ```
 
 
-##### Reach out to **[/pranavgore09](https://github.com/pranavgore09)** in case things are not working as expected
+##### Questions? contact **[/pranavgore09](https://github.com/pranavgore09)**
```

Comment:
Shouldn't we say that user are supposed to open an issue instead of contacting an individual? I'm happy to then assign all minishift issues to you @pranavgore09 but by having issues (maybe even invalid ones) will help us track problems much better IMHO.

Reply:
right, done in https://github.com/fabric8-services/fabric8-wit/pull/1792/commits/1c28230ac10a04dfd57418bae7eeb93f538445a2

Reply:
or maybe point them to a chat/channel as well ? because sometimes folks wish to talk synchronously :)

Reply:
Sure!
added in https://github.com/fabric8-services/fabric8-wit/pull/1792/commits/9666e5cd6ac2d0ba1f6b629de01beb8a007fe3eb